WordPress/Classes/Extensions/NSDate+Helpers.swift

```diff
@@ -20,6 +20,16 @@ extension Date {
             formatter.timeZone      = TimeZone(secondsFromGMT: 0)
             return formatter
         }()
+
+        static let pageSectionFormatter: TTTTimeIntervalFormatter = {
```

Comment:
I think the goal was having all the formatters in the same place, so it's easier to spot all the different ways we're showing dates across the app.